docs/redisearch.md

```diff
@@ -49,18 +68,37 @@ fields.put("price", 1337);
 client.hset("item:hw", RediSearchUtil.toStringMap(fields));
 ```
 
+Another way to insert documents:
+
+```java
+client.hsetObject("item:hw", fields);
+```
+
+### Querying
+
 Searching the index:
 
 ```java
-// creating a complex query
 Query q = new Query("hello world")
         .addFilter(new Query.NumericFilter("price", 0, 1000))
         .limit(0, 5);
 
-// actual search
 SearchResult sr = client.ftSearch("item-index", q);
+```
+
+Alternative searching using FTSearchParams:
```

Comment:
In the field-level decoding PR we will need to change it to:

> Alternatively, you can use FTSearchParams to access more advanced capabilities, such as field-level decoding control. Here is an example:

Reply:
@uglide IMO, we need a new example. I'm having difficulty making a decent example from title (text), body (text) and price (numeric) fields.